### PR TITLE
Added brandwatch client users endpoints

### PR DESCRIPTION
**Pull request by benSlaughter**
### Details
- Added GET /brandwatch/clients/{client_id}/users endpoint
- Added GET /brandwatch/clients/{client_id}/users/{user_id} endpoint
- Added PUT /brandwatch/clients/{client_id}/users/{user_id} endpoint

### CODE REVIEW

**Comment by benSlaughter** [comment]
Coverage is included, that's nice :)

File: lib/bwapi/client/brandwatch/clients/users.rb [hunk 1]
```diff
@@ -4,6 +4,13 @@ module Brandwatch
       module Clients
         # Users module for brandwatch/users endpoints
         module Users
+          # List all users for a client
```
Comment (jonathanchrisp) [inline lib/bwapi/client/brandwatch/clients/users.rb]: Small change, I think we should change this to be: `List all users in a client`. If we could change the methods below that would be great.

**Comment by jonathanchrisp** [comment]
LGTM - one minor comment

**Comment by benSlaughter** [comment]
@jonathanchrisp Updated :)

**Comment by jonathanchrisp** [comment]
LGTM so merging

**Pull request merged**